docs/getting-started-guides/docker-multinode.md

```diff
@@ -1 +1 @@
-# Stop. This guide has been superseded by [kubeadm](../kubeadm/). This page is only present for historical purposes.
+# Stop. This guide has been superseded by [kubeadm](/docs/admin/kubeadm/). This page is only present for historical purposes.
```

Comment:
```
**Caution:** This guide is deprecated and replaced by [kubeadm](/docs/admin/kubeadm/).
{: .caution}
```

Reply:
Makes sense - will update! In either case, I think this kind of updated is warranted, as the referenced document that is replacing the this moved due to the change in relative location.